### PR TITLE
Also install 64-bit versions of dlls from win7sp1

### PR DESCRIPTION
**Pull request by gFreezer**
Mainly desireable to make 64-bit applications look less ugly by installing native gdiplus

### CODE REVIEW

**Comment by winetaste** [comment]
Nice! 
Btw.: the common 64-bit WINEPREFIX warning doesn't say that winetricks only installs 32-bit version of a dll. There should be an extra hint for this.

**Comment by austin987** [comment]
@winetaste, that should be its own issue. Send a pr or open an issue, please.

File: src/winetricks [hunk 1]
```diff
@@ -4675,6 +4675,16 @@ helper_win7sp1()
     w_try_cabextract -d "$W_TMP" -L -F "$filename" "$W_CACHE"/win7sp1/windows6.1-KB976932-X86.exe
 }
 
+helper_win7sp1_x64()
+{
+    filename=$1
+
+    # https://www.microsoft.com/en-us/download/details.aspx?id=5842
+    w_download_to win7sp1 https://download.microsoft.com/download/0/A/F/0AFB5316-3062-494A-AB78-7FB0D4461357/windows6.1-KB976932-X64.exe 74865ef2562006e51d7f9333b4a8d45b7a749dab
```
Comment (vszakats) [inline src/winetricks]: sha256 would be welcome here:
`f4d1d418d91b1619688a482680ee032ffd2b65e420c6d2eaecf8aa3762aa64c8`

**Comment by austin987** [comment]
@winetaste 32-bit dll warning added here:
https://github.com/Winetricks/winetricks/commit/efb178992adefe84dfcfcea04ffdb42d1b6e684a

**Comment by austin987** [comment]
https://github.com/Winetricks/winetricks/commit/c0618e07b3bde15e33faa0f95e76f96378c0dc74
https://github.com/Winetricks/winetricks/commit/6cad012a1e66c20fad559bf3341228792cc839a3
https://github.com/Winetricks/winetricks/commit/0dea0c9652e129cab9a00ccb9a1b497768b99172
https://github.com/Winetricks/winetricks/commit/0b1708a9bc67d8f405eb8dbb30d19b4f486c8504
https://github.com/Winetricks/winetricks/commit/98f15be400a2cbce7624ace4114d257b5470f860

**Pull request closed without merging**